go.mod

```diff
@@ -6,46 +6,49 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.2.0
 	github.com/hashicorp/terraform-plugin-sdk v1.14.0
-	github.com/rancher/norman v0.0.0-20210225010917-c7fd1e24145b
-	github.com/rancher/rancher v0.0.0-20210714221037-3c54189441fd
+	github.com/rancher/norman v0.0.0-20210608202517-59b3523c3133
+	github.com/rancher/rancher v0.0.0-20210830223634-df2432ad895c
 	github.com/rancher/rancher/pkg/apis v0.0.0
 	github.com/rancher/rancher/pkg/client v0.0.0
-	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
+	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
 	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
-	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.20.6
-	k8s.io/apimachinery v0.20.6
-	k8s.io/apiserver v0.20.6
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.21.2
+	k8s.io/apimachinery v0.21.2
+	k8s.io/apiserver v0.21.0
```

Comment:
We should probably use the same patch version. Thoughts?

Reply:
I usually see the "replace" keyword and its block before the "require" keyword and its block... Disregard this comment. 
![image](https://user-images.githubusercontent.com/39320683/131879180-ca0b0e58-9b42-469a-883a-97c5ee7726e2.png)